### DISCUSSION TITLE
Kolo TV [MG]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#396] [Show and tell]
### Channel ID

KoloTV.mg

### Stream URL

http://free.fullspeed.tv/iptv-query?streaming-ip=https://twitch.tv/kolotvlive

### Notes

Not 24/7

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://twitch.tv/kolotvlive (offline)
